elasticdl/docker/Dockerfile.dev_allreduce

```diff
@@ -1,12 +1,14 @@
 FROM elasticdl:dev
 
 WORKDIR /root/
-RUN git clone --depth=1 https://github.com/caicloud/ftlib.git
-RUN cd /root/ftlib && pip install -r requirements.txt
 
 # Note that pip is having issue downloading PyTorch on manylinux so we use curl to download it instead
-RUN curl -o torch-1.4.0-cp36-cp36m-manylinux1_x86_64.whl https://files.pythonhosted.org/packages/24/19/4804aea17cd136f1705a5e98a00618cb8f6ccc375ad8bfa437408e09d058/torch-1.4.0-cp36-cp36m-manylinux1_x86_64.whl
-RUN pip install torch-1.4.0-cp36-cp36m-manylinux1_x86_64.whl
+RUN curl -o torch-1.4.0-cp36-cp36m-manylinux1_x86_64.whl \
```

Comment:
- Use `curl --silent` to reduce the printing of progress.  This saves the messy output from `docker build`.
- Use `curl -L` to support HTTP redirection, which is common in many download sites.
- Use `curl -O` to use the filename in the URL.

```
curl -sLO https://files.pythonhosted.org/packages/24/19/4804aea17cd136f1705a5e98a00618cb8f6ccc375ad8bfa437408e09d058/torch-1.4.0-cp36-cp36m-manylinux1_x86_64.whl
```